test/fixtures/payloads/basic/index.ts

```diff
@@ -0,0 +1,12 @@
+import prGeneric from "./pr.json";
+
+const prOpenedPayload: Record<string, unknown> = prGeneric;
+prOpenedPayload["action"] = "opened";
+
+const prReopenedPayload: Record<string, unknown> = prGeneric;
+prOpenedPayload["action"] = "reopened";
+
+const prSynchronizePayload: Record<string, unknown> = prGeneric;
+prOpenedPayload["action"] = "synchronize";
+
+export { prOpenedPayload, prReopenedPayload, prSynchronizePayload};
```

Comment:
Codacy found an issue: [file should end with a newline](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6091206)